drivers/sensor/vishay/vcnl36825t/vcnl36825t.c

```diff
@@ -242,26 +242,10 @@ static int vcnl36825t_init_registers(const struct device *dev)
 		reg_value |= VCNL36825T_PS_IT_1T;
 		data->meas_timeout_us *= 1 * VCNL36825T_FORCED_FACTOR_SCALE;
```

Comment:
You may remove `VCNL36825T_FORCED_FACTOR_SCALE` since it was necessary to get the 0.5 values into the integer range. If you do so, you can also simplify `VCNL36825T_FORCED_FACTOR_SCALE * VCNL36825T_FORCED_FACTOR_SCALE` in line 380 to a simple `VCNL36825T_FORCED_FACTOR_SCALE`

Reply:
Thanks. Addressed.